.github/workflows/test-cirros-bionic.yaml

```diff
@@ -0,0 +1,199 @@
+name: Test CirrOS build
+
+on:
+  pull_request:
+    types:
+      - opened
```

Comment:
What about updated PRs?

Reply:
drop 'types' part

Reply:
> drop 'types' part

@hrw @osfrickler Regarding pull requests, do we want to have triggers for the 0.5 and 0.6 branches? This would allow us to cherry-pick commits that we want to backport to stable versions of CirrOS (i.e. backport a fix for RSA2 to the 0.5 branch). This would also allow us to have specific patches for each branch/release.

Reply:
I would like to merge CI before creating 0.5 branch. This way we will have CI defined for "legacy" 0.5 branch and for anything newer.